src/LoginPage/LoginForm.jsx

```diff
@@ -1,6 +1,15 @@
 import { memo } from 'react';
 
-export default memo(({ isLogin, onClick, onChange }) => {
+export default memo(({
+  isLogin,
+  fields,
+  onClick,
+  onChange,
+}) => {
+  const {
+    email, password, firstName, lastName,
+  } = fields;
```

Comment:
```suggestion
export default memo(({
  isLogin,
  fields: { email, password, firstName, lastName },
  onClick,
  onChange,
}) => {
```
이런식으로 작성해도 편하더라구요 ㅎㅎ 참고하세용!~

Reply:
앗 그렇군요! 어차피 아래에서 한번더 구조분해할당을 사용해야하는데
바로 사용할 수 있겠군요!
생각하지 못한 부분입니다! 감사합니당 🙏🏼